app/display.py

```diff
@@ -0,0 +1,23 @@
+from app.book import Book
+
+
+class BookDisplay:
+    @staticmethod
+    def display(book: Book, display_type: str) -> None:
+        if display_type == "console":
```

Comment:
The BookDisplay class is not properly adhering to Open-Closed principle. Instead of hardcoding the display types within the method, it would be better to utilize polymorphism or a strategy pattern to handle different display types. This way new display types can be added without modifying the existing code.